addons/bitmap-copy/userscript.js

```diff
@@ -14,5 +13,8 @@ export default async ({ addon, console }) => {
       .copyImage(dataURL)
       .then(() => console.log("Image successfully copied"))
       .catch((e) => console.error(`Image could not be copied: ${e}`));
-  });
+  };
+  addon.tab.redux.addEventListener("statechanged", handleCopy);
+  addon.self.addEventListener("disabled", () => addon.tab.redux.removeEventListener("statechanged", handleCopy));
+  addon.self.addEventListener("reenabled", () => addon.tab.redux.addEventListener("statechanged", handleCopy));
```

Comment:
couldn't you have just added an if check to return if it's disabled? there are a lot of other examples of this below.

Reply:
Simply returning is recommended as long as it produces cleaner code. No need to worry about memory leaks at this scale.